packages/keystore/lib/models/privatekeys.ts

```diff
@@ -152,3 +152,15 @@ function decryptValue(encryptedValue: Buffer): Result<string, PrivateKeyError> {
     }
     return Ok(encryption.decrypt(encrypted, iv, tag));
 }
+
+export async function deleteExpiredPrivateKeys(db: knex.Knex, { limit, deleteOlderThan }: { limit: number; deleteOlderThan: number }): Promise<number> {
```

Comment:
`deleteOlderThan` could be `olderThan` since it is a parameter of a function called `delete...`